Configurations/windows-makefile.tmpl

```diff
@@ -460,10 +465,13 @@ $target: $deps "$ordinalsfile" "$mkdef_pl"
 	DEL $shlib.def.tmp
 	"\$(PERL)" "$mkrc_pl" $shlib$shlibext > $shlib.rc
 	\$(RC) \$(RCOUTFLAG)$shlib.res $shlib.rc
+	IF EXIST $shlib$shlibext.manifest DEL /F /Q $shlib$shlibext.manifest
 	\$(LD) \$(LDFLAGS) \$(LIB_LDFLAGS) \\
 		/implib:\$@ \$(LDOUTFLAG)$shlib$shlibext /def:$shlib.def @<< || (DEL /Q \$(\@B).* $shlib.* && EXIT 1)
 $objs $shlib.res$linklibs \$(EX_LIBS)
 <<
+	IF EXIST $shlib$shlibext.manifest \\
+	   \$(MT) \$(MTFLAGS) \$(MTINFLAG) $shlib$shlibext.manifest \$(MTOUTFLAG)$shlib$shlibext
```

Comment:
Given appearance of `$target{mtinflag}` above line is probably meant to read `\$(MTINFLAG)$shlib$shlibext.manifest`, i.e. without space to cover for hypothetical case when it would have to look like `-in:this-n-that`.

One can also wonder if it would be appropriate to delete manifest file after it's embedded into binary. Rationale is that It would make test pass more "realistic", i.e. closer to how it would look like when installed...

On side note I for one find parameterization a bit excessive in this case. I mean I wouldn't feel bad about spelling command line in "clear text" as `mt -nologo -manifest ...`, as opposite to introducing more configuration parameters :-)